types/ws/index.d.ts

```diff
@@ -114,69 +116,78 @@ declare class WebSocket extends EventEmitter {
     // HTML5 WebSocket events
     addEventListener<K extends keyof WebSocket.WebSocketEventMap>(
         type: K,
-        listener: (event: WebSocket.WebSocketEventMap[K]) => void,
+        listener:
+            | ((event: WebSocket.WebSocketEventMap[K]) => void)
+            | { handleEvent(event: WebSocket.WebSocketEventMap[K]): void },
         options?: WebSocket.EventListenerOptions,
     ): void;
     removeEventListener<K extends keyof WebSocket.WebSocketEventMap>(
         type: K,
-        listener: (event: WebSocket.WebSocketEventMap[K]) => void,
+        listener:
+            | ((event: WebSocket.WebSocketEventMap[K]) => void)
+            | { handleEvent(event: WebSocket.WebSocketEventMap[K]): void },
```

Comment:
`@types/node` defines [`EventListenerObject`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3002c74ef875de9c5c4992723591eccc2c6eb42e/types/node/dom-events.d.ts#L100-L102), but that interface is not generic, so I think this solution is the best